exercises/concept-wip/crystal-hunter/.meta/src/exemplar.cr

```diff
@@ -0,0 +1,17 @@
+class Rules
+  def eat_ghost?(power_pellet_active, touching_ghost)
+    power_pellet_active && touching_ghost
+  end
```

Comment:
All the tests and example solution needs to be rewritten to actually use the new non-pacman terminology that you used in the instructions. After you do that, I will be able to follow the instructions and check if they make sense by solving the exercise myself.